packages/react/src/button/button.test.tsx

```
@@ -0,0 +1,32 @@
+import { render, screen, fireEvent } from "@testing-library/react";
```

ESLint

[error] Unable to resolve path to module '@testing-library/react'.

(import-x/no-unresolved)